modules/news/news.service.js

```diff
@@ -15,22 +18,25 @@ class NewsService {
   }
 
   async addNews(data) {
-    console.log('data', data.title[0], data.title[1]);
+    if (await this.checkNewsExist(data)) {
+      throw new Error(NEWS_ALREADY_EXIST);
+    }
+    return new News(data).save();
+  }
+
+  async deleteNews(id) {
+    return await News.findByIdAndDelete(id);
+  }
+
+  async checkNewsExist(data) {
     const news = await News.find({
```

Comment:
Use the method `count` here to reduce the amount of data that is transferred between app and db